backendjs/server/api/controllers/event/controller.js

```diff
@@ -0,0 +1,182 @@
+const Model = require('../../../sequelize/models');
+const Event = Model.Event;
+const EventParticipation = Model.EventParticipation;
+export class Controller {
+  async getOne(req, res) {
+    try {
+      const { id } = req.params;
+      let event = await Event.findOne({
+        where: {
+          id: id
+        }
+      });
+      let userevents = await EventParticipation.findOne({
+        where: {
+          user: req.user.sub,
+          event: id
+        }
+      });
+      if (event === null) {
+        res.status(400).json({
+          success: false,
+          message: 'Group does not exist'
+        });
+      } else {
+        let participate = true;
+        if (userevents == null) {
+          participate = false;
+        }
+        res.status(200).json({
+          success: true,
+          message: 'Successful',
+          data: {
+            id: id,
+            name: event.name,
+            description: event.description,
+            tags: event.tags,
+            owner: event.owner,
+            next: event.next,
+            datetime: event.datetime,
+            length: event.length,
+            image: event.image,
+            participating: participate
+          }
+        });
+      }
+    } catch (error) {
+      res.status(500).json({
+        success: false,
+        messasge: error.message
+      });
+    }
+  }
+
+  async get(req, res) {
+    try {
+      let events = await Event.findAll();
+      let userevents = await EventParticipation.findAll({
+        where: {
+          user: req.user.sub
+        }
+      });
+      let eventlist = [];
```

Comment:
similar map methods from https://github.com/summerprogram2019/limber/pull/5#discussion_r302008292 can be used here